src/main/java/org/maxgamer/rs/cache/Cache.java

```diff
@@ -14,8 +14,77 @@
 
 import org.maxgamer.rs.cache.reference.Reference;
 import org.maxgamer.rs.cache.reference.ReferenceTable;
+import org.maxgamer.rs.structure.configs.FileConfig;
+import org.maxgamer.rs.util.Log;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AUndocumentedApi)